.coveragerc

```diff
@@ -1,3 +1,3 @@
 [run]
 source = .
-omit = .eggs/*,.tox/*,*tests*,setup.py
+omit = .eggs/*,.tox/*,*tests*,setup.py,.direnv/*,.venv/*,venv/*,v0x01/match.py
```

Comment:
Since we're planning to drop OF 1.0 support in the future, I figured, its code cov shouldn't matter for our team in terms of code quality, on this module there were several funcs/conditionals to be covered, I excluded it from code cov.